flux-typeck/src/dbg.rs

```diff
@@ -18,6 +18,26 @@ macro_rules! _check_span {
 }
 pub use crate::_check_span as check_span;
 
+#[macro_export]
+macro_rules! _check_top_span {
```

Comment:
We really need a better name for this function, check is too generic, but can't think of anything else.

Reply:
Neither could I. As long as nothing downstream depends on the terminology we can change it later.